.gitignore

```diff
@@ -109,3 +109,6 @@ pkg
 CMakeUserPresets.json
 bld.rippled/
 .vscode
+
+# Suggested in-tree build directory
+/.build/
```

Comment:
the prefix `/` limits the exclude to the top-level of the worktree, in case there are directories named`.build` elsewhere in the project with entirely different semantics. Happy either way, but it is very precise this way.